src/usdb_syncer/usdb_scraper.py

```diff
@@ -335,8 +335,8 @@ def _parse_details_table(
         cover_url=cover_url,
         bpm=details_table.find(string="BPM").next.text,  # type: ignore
         gap=details_table.find(string="GAP").next.text,  # type: ignore
-        golden_notes=details_table.find(string=usdb_strings.GOLDEN_NOTES).next.text,  # type: ignore
-        song_check=details_table.find(string=usdb_strings.SONGCHECK).next.text,  # type: ignore
+        golden_notes=details_table.find(string=usdb_strings.GOLDEN_NOTES).next.text.rstrip() == usdb_strings.YES,  # type: ignore
+        song_check=details_table.find(string=usdb_strings.SONGCHECK).next.text.rstrip() == usdb_strings.YES,  # type: ignore
```

Comment:
I guess black doesn't complain, because it can't refactor these lines due to the comment, but they are still too long.
Maybe we could add a type safe helper for the `details_table.find(string=...).next.text` part, which comes up a lot.

Reply:
What is a type safe helper?